charts/spire/charts/spiffe-csi-driver/templates/daemonset.yaml

```diff
@@ -68,7 +76,7 @@ spec:
           args: [
             "-csi-address", "/spiffe-csi/csi.sock",
             "-kubelet-registration-path", "/var/lib/kubelet/plugins/csi.spiffe.io/csi.sock",
-            "-health-port", "9809"
+            "-health-port", {{ .Values.nodeDriverRegistrar.healthPort | quote }}
```

Comment:
What is the value of being able to change this? It just makes the helm chart complexer for the end user if there is no specific usecase to change this.

Reply:
We use the Amazon EFS CSI Driver and it's daemonset uses this port for its healthPort (https://github.com/kubernetes-sigs/aws-efs-csi-driver/blob/master/charts/aws-efs-csi-driver/values.yaml#L124).

This _may_ be a non-issue if we can use the namespaceSelector to exclude it.